src/openrct2-ui/windows/Ride.cpp

```diff
@@ -5826,7 +5830,44 @@ static void window_ride_graphs_scrollpaint(rct_window *w, rct_drawpixelinfo *dpi
             top = bottom;
             bottom = tmp;
         }
-        gfx_fill_rect(dpi, x, top, x, bottom, x > measurement->current_item ? PALETTE_INDEX_17 : PALETTE_INDEX_21);
+
+        // If the lateral or vertical forces are too extreme, make line red.
+        bool colourIsRed = false;
```

Comment:
Nitpicking, but perhaps a nice idea to make this variable name a bit more descriptive `gforceAboveThreshold` or something.